syntax = "proto3";

message ImageRequest {
  int32 width = 1;
  int32 height = 2;
  bytes data = 3;
}

message MaskReply{
    int32 length = 1;
    bytes data = 2;
}

service InferenceSam {
  rpc Inference(ImageRequest) returns (MaskReply){}
}
